detection-rules/attachment_malicious_onenote_commands.yml

```
@@ -0,0 +1,37 @@
+name: "Attachment: Malicious OneNote Commands"
+description: |
+  Scans for OneNote attachments that contain suspicious commands that may indicate malicious activity.
+references:
+  - "https://www.trustedsec.com/blog/new-attacks-old-tricks-how-onenote-malware-is-evolving/"
+type: "rule"
```

```suggestion
type: "rule"
authors:
  - twitter: "Kyle_Parrish_"
    name: "Kyle Parrish"
```

don't forget to give yourself credit :)